packages/app/src/systems/Account/components/EthAddress/EthAddress.tsx

```diff
@@ -0,0 +1,37 @@
+import type { ThemeUtilsCSS } from '@fuel-ui/css';
+import { cssObj } from '@fuel-ui/css';
+import { Copyable, Flex, Text, Tooltip } from '@fuel-ui/react';
+import { bn } from 'fuels';
+
+import { shortAddress } from '~/systems/Core';
+
+export type EthAddressProps = {
+  address: string;
+  css?: ThemeUtilsCSS;
+};
+
+export const EthAddress = ({ address, css }: EthAddressProps) => {
+  const ethAddress = bn(address).toHex(20);
```

Comment:
Would be good to have a address validation here, similar to what we do in [`FuelAddress`](https://github.com/FuelLabs/fuels-wallet/blob/master/packages/app/src/systems/Account/components/FuelAddress/FuelAddress.tsx#L14) component